proposals/2020/Add Simple Impact Analysis.md

```diff
@@ -0,0 +1,91 @@
+# Add Simple Impact Analysis
+
+## Overview
+
+This proposal is to add the `auc:SimpleImpactAnalysis` child element to the `auc:PackageOfMeasures` element. It also includes the suggestion that we move some of the direct children of `auc:PackageOfMeasures` into a new element `auc:ComplexImpactAnalysis`.
+
+## Justification
+
+Standard 211 6.1.5.c-6.1.5.g as well as 6.1.6.d-6.1.6.g requires a summary of the impact of each recommended measure. Specifically, it states:
+> c. Impact on occupant comfort (improved thermal comfort, indoor air quality [IAQ], lighting quality, acoustics)
+> d. Estimated cost (high, medium, low)
+> e. Estimated level (high, medium, low) of annual savings
+> f. Estimated level (high, medium, low) of return on investment (ROI)
+> g. Priority (high, medium, low)
+
+Additionally, Standard 211 is explicit in discriminating the "Low-Cost and No-Cost" recommendations (section 6.1.5) from the "Capital" recommendations (section 6.1.6), so it might not be a bad idea to add a category to specify this.
```

Comment:
I think it can be argued that this discrimination is unnecessary/redundant because we apready specify the estimated cost (high, medium, low). Something to consider.